src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -10,8 +10,8 @@
         public static final boolean DRIVE_ENABLED = true;
         public static final boolean DRIVER_VIS_ENABLED = false;
         public static final boolean GOAL_VIS_ENABLED = false;
-        public static final boolean INDEX_ENABLED = false;
-        public static final boolean INTAKE_ENABLED = false;
+        public static final boolean INDEX_ENABLED = true;
+        public static final boolean INTAKE_ENABLED = true;
```

Comment:
These don't actually work since the motor CAN ids are all 0, so I'd recommend leaving them at 'false' for the main code.